Cargo.lock

```diff
@@ -144,15 +144,6 @@ version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d92bec98840b8f03a5ff5413de5293bfcd8bf96467cf5452609f939ec6f5de16"
 
-[[package]]
```

Comment:
Oh nice, so many removed dependencies. This removes our total dependencies from 354 to 335!